### PR TITLE
Update dir2html.py

### PR DESCRIPTION
**Pull request by bgruening**
Python3 porting.

### CODE REVIEW

File: tools/macs2/dir2html.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
 #!/usr/bin/env python
```
Comment (nsoranzo) [inline tools/macs2/dir2html.py]: If you want to maintain Python2 compatibility you can add here:
```python
from __future__ import print_function
```

File: tools/macs2/dir2html.py [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 import sys
 from xml.sax.saxutils import escape
 
 
-def make_table( directory ):
+def make_table(directory):
     ret = ['<table class="fileList">\n']
-    for file in os.listdir( directory ):
-        ret.append('<tr><td class="file"><a href="%s">%s</a></td></tr>\n' % ( file, escape(file).replace( 'MACS2_', '' ) ))
+    for file in os.listdir(directory):
```
Comment (nsoranzo) [inline tools/macs2/dir2html.py]: `file` is a Python reserved word, can you change it to something else?

File: tools/macs2/dir2html.py (hunk 1 above)
Comment (bgruening) [inline tools/macs2/dir2html.py]: Python2.7 does support the brackets, isn't it? And we do only support Python2.7. I added it nevertheless.

Comment (nsoranzo) [inline tools/macs2/dir2html.py]: `print(foo)` in Py2.7 should be read as `print (foo)`, i.e. `print` is not a function and the parentheses are interpreted as defining the order of operations. This breaks if e.g. a comma is added inside the parentheses.

Python3:
```
In [1]: print ('foo', 'bar')
foo bar
```
Python2:
```
In [1]: print ('foo', 'bar')
('foo', 'bar')
```

So I suggest to always add the `print_function` import.

**Pull request merged**